components/extractors/other/telemetry-data/telemetry-data.md

```diff
@@ -0,0 +1,292 @@
+---
+title: Telemetry Data
+permalink: /components/extractors/other/telemetry-data/
+redirect_from:
+    - /extractors/other/telemetry-data/
+
+---
+
+* TOC
+{:toc}
+
+The Telemetry Data extractor allows you to retrieve data about your project or about your whole organization. 
+It helps you monitor activities and usage of your Keboola Connection projects. It also helps Keboola calculate your project consumption.
+
+## Configuration
+To configure the extractor, select one of the following modes:
+ 
+1. **Project mode** --- extracts data from a selected Keboola Connection project only.
+2. **Organization mode** --- extracts data from all projects of your organization. The data is extracted 
+to a single target project that must be set via your Keboola Connection Success Manager.
+
+## Data Model
+The model below helps you better understand relations between individual tables extracted by this component.
+
+*Note: The tables `contact_limit_monthly`, `kbc_organization`, and `usage_metric` are extracted only in **Organization** mode.*
+
+<iframe width="100%" height="400" src='https://dbdiagram.io/embed/602629a380d742080a3a406a'> </iframe>
```

Comment:
tohle vložit jako SVG, kdyžtak s linkem na dbdiagram, ale nebudem vkládat 3rd party iframe

Reply:
Link zde: https://dbdiagram.io/d/602629a380d742080a3a406a
@hhanova Máš jak převést to png, nebo mám někde hledat řešení?

Reply:
nemusí se to převádět na PNG, může zůstat jako SVG, ale hostovaný u nás

Reply:
tak ja zatim dam PNG, protoze nevim, jak poridit SVG  :-)